test/specs/rules/disallow-space-before-semicolon.js

```diff
@@ -0,0 +1,31 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/disallow-space-before-semicolon', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+    });
+
+    it('should not accept a false value as an option', function() {
+        assert.throws(function() {
+            checker.configure({ disallowSpaceBeforeSemicolon: false });
+        });
+    });
+
+    it('does not allow spaces before semicolons', function() {
+        checker.configure({ disallowSpaceBeforeSemicolon: true });
+
+        assert(!checker.checkString('; ;').isEmpty());
+        assert(!checker.checkString('var a = 1 ;').isEmpty());
+        assert(!checker.checkString('var a = 2  ;').isEmpty());
```

Comment:
Maybe it's old, but I think it makes sense to be specific when possible (even though I guess it's not a big deal)? It doesn't seem to be used that often and only when there is a large group of tests. Maybe we should change all those? 

Don't know the reason other than it's easier to type haha.